Src/LinearSolvers/MLMG/AMReX_MLMG.H

```diff
@@ -217,8 +217,8 @@ private:
 #else
 
     // Hypre::Interface hypre_interface = Hypre::Interface::structed;
-    Hypre::Interface hypre_interface = Hypre::Interface::semi_structed;
-    // Hypre::Interface hypre_interface = Hypre::Interface::ij;
+    // Hypre::Interface hypre_interface = Hypre::Interface::semi_structed;
+    Hypre::Interface hypre_interface = Hypre::Interface::ij;
```

Comment:
@WeiqunZhang we can get rid of the `#ifdef AMREX_USE_EB` and just have one definition?